qp_deblur/deblur.py

```diff
@@ -321,6 +321,16 @@ def generate_insertion_trees(placements, out_dir,
                      % (file_ref_rename, std_out, std_err))
         raise ValueError(error_msg)
 
+    # this is required cause there is a missing branch in the SEPP tree
+    new_tree = None
+    with open(file_tree, 'r') as tree_fp:
+        tree = tree_fp.read()
+        if tree.endswith(tuple(["'k__Bacteria');", "'k__Bacteria');\n"])):
```

Comment:
this works for our current issue, but is not very general. Other trees, e.g. Silva, might have other names for their root nodes but also missing branch length. I used a very general, but slow approach for the q2 plugin: https://github.com/biocore/q2-fragment-insertion/blob/0ee397c308179279c855e50838afdc18a1fc293c/q2_fragment_insertion/_insertion.py#L78
It's up to the Qiita maintainer to decide what to use here

Reply:
K, this approach is safer, will change.